README.md

```diff
@@ -42,6 +42,12 @@ Note that step-by-step bootstrapping instructions are given in the
 account-specific `README.md` files found in the account
 subdirectories.
 
+## Notes ##
+
+Running `pre-commit` requires running `terraform init` in every directory that
+contains Terraform code. In this repository, these are the main directory and
+every directory under `examples/`.
```

Comment:
This blurb be updated to something like:
In this repository, this includes each account directory (e.g. `audit/`, `dns/`, `dynamic/`, etc.), as well as `provisionaccount-role-tf-module` and every directory under its `examples/` directory.

Reply:
See commit e0f8b74.